README.md

```diff
@@ -16,6 +16,18 @@ in your tests without having to directly interact with the DOM, and the chai
 plugin can be used to write readable tests with understandable output for
 mounted Vue components.
 
+> NOTE: vue-test requires the **full** version of Vue (which includes the compiler).  Make sure your build configuration for testing aliases `vue` properly.  For example (webpack 2):
+
+```javascript
+{
+  resolve: {
+    alias: {
+      'vue$': 'vue/dist/vue.esm.js'
```

Comment:
Is there any reason this is using vue.esm.js instead of vue.js? (I'll admit, I'm not too familiar with the difference)